docs/conf.py

```diff
@@ -65,6 +65,11 @@
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
+# -- Extension options -------------------------------------------------------
+
+myst_enable_extensions = [
+    "colon_fence",
+]
```

Comment:
This is not strictly related to the admonitions, right?
Wondering why did you include it? Are you planning to bring other extensions in the future?

Reply:
oops good catch - forgot to document it. Was fixing a small bug where we didn't enable this syntax in the myst parser (see the last part of this section: https://pydata-sphinx-theme.readthedocs.io/en/latest/contribute/setup.html#setup-pre-commit)

Reply:
> forgot to document it

Gotcha! Can you maybe add a new commit with tiny docs so the change does not feel totally disconnected?

Reply:
done - what do you think about the comment?